browser/ui/views/brave_actions/brave_actions_container.cc

```diff
@@ -307,6 +311,25 @@ void BraveActionsContainer::OnExtensionActionUpdated(
 }
 // end ExtensionActionAPI::Observer
 
+// BraveActionAPI::Observer
+void BraveActionsContainer::OnBraveActionShouldTrigger(
+    const std::string& extension_id,
+    std::unique_ptr<std::string> ui_relative_path) {
+  if (!IsContainerAction(extension_id)) {
+    return;
+  }
+  if (actions_[extension_id].view_controller_) {
+    // auto view_controller = static_cast<BraveActionViewController*>(
```

Comment:
Can this comment be removed or does it indicate needed code?

Reply:
Thanks for the spot - removed